getting-started: Show people how to search a simple pattern (#744)
returntocorp/semgrep-docs: docs/getting-started.md

        ```sh
        docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep --config=auto
        ```

> 2. Confirm installation by running a simple search pattern in the current directory. To do so, run the following command:

Author: ```suggestion
2. Confirm installation by running a simple search pattern. For example, run the following command:
```

    ```sh

>     semgrep --pattern '1' --lang generic

Author: ```suggestion
    semgrep --pattern '127.$A.$B.$C' --lang generic /etc/hosts
```

This command might be more reliable. If a user ran the original query in their home directory, semgrep would keep scanning for minutes and yield hundreds of thousands of results.
